picturepoint/src/Login/components/oldNavigation.js

```diff
@@ -1,9 +1,11 @@
+//NOT USING COMPONENT
```

Comment:
It's Haocheng's old navigation component. I left it in the repo in case we wanted to go back. I noted the component with that comment so we can delete it later on.

Reply:
Perf !